ui/apps/platform/src/services/CloudSourceService.ts

```diff
@@ -0,0 +1,39 @@
+import axios from './instance';
+import { Empty } from './types';
+
+export type CloudSourceType = 'TYPE_UNSPECIFIED' | 'TYPE_PALADIN_CLOUD' | 'TYPE_OCM';
+
+export type CloudSourceCredentials = {
+    secret: string;
+};
+
+export type PaladinCloudConfig = {
+    endpoint: string;
+};
+
+export type CloudSourceIntegration = {
+    id: string;
+    name: string;
+    type: CloudSourceType;
+    credentials: CloudSourceCredentials;
+    skipTestIntegration: boolean;
+    paladinCloud?: PaladinCloudConfig;
+};
+
+const cloudSourcesURL = `/v1/cloud-sources`;
+
+export function fetchCloudSources(): Promise<{
+    response: { cloudSources: CloudSourceIntegration[] };
+}> {
+    return axios.get(cloudSourcesURL).then((response) => ({
+        response: response.data,
+    }));
+}
+
+export function deleteCloudSource(id: string): Promise<Empty> {
+    return axios.delete(`${cloudSourcesURL}/${id}`);
```

Comment:
For a follow up contribution after you merge this, can you add `.then((response) => response.data)` to chain.

As written the function returns (leaks) the entire response object instead of just the `data` empty object.